src/UI/Component/Dropzone/File/Dropzone.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Interface Dropzone
+ *
+ * Describes a dropzone that can be configured to use the darkened background
+ * or not.
```

Comment:
Thanks @alex40724 for sharing this observation. Concerning the darkened comment: I think you do have a valid point there. @BilledTrain380 do you have a less variable to set the highlighted background? Further I think we need some rule to decide when we use the highlighted background version and when not. Can we do that on a generic dropdown level? Do we even need both?

Concerning the hierarchy: The hierarchy path visible through the factory is 
`$f->dropzone()->file()->standard().`

@BilledTrain380 : Why did you place the dropzone interface on the file level? is this not the valid interface for all dropzones (as alex suggests for colors in his comment)? If you have an interface just valid for just the file dropzones, I would rather call it file.